messagebox.py

```diff
@@ -0,0 +1,19 @@
+import wx
+ 
+class AlertBox():
+    def __init__(self, text):
+        self.text = text
+
+    def show(self, event):
+         wx.MessageBox(self.text, 'Upozornenie', wx.OK | wx.ICON_WARNING)

```

Comment:
tu by som to spravil tak aby text nedostavala trieda pri vytvarani ale aby na to bola metoda, potom by handler nemusel vytvarat novu instanciu pre kazdu spravu

Reply:
upravené, je to ok?

Reply:
ano, uz som to mergeol, len nabuduce poprosim uploadovat kod bez testovacich vypisov (aj zakomentovanych)